prosopopee/themes/exposure/templates/index.html

```diff
@@ -6,7 +6,18 @@
 {% endblock %}
 
 {% block logo %}
-    <img id="logo" src="static/img/logo.svg" />
+    {% if sub_index %}
+    <div class="back-to-home">
+        <hr>
+        <a href="../">
```

Comment:
Is it suposed to go back one level or to home? If it's to home this won't work for sub sub galleries.